.circleci/unittest/windows/scripts/set_cuda_envs.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+if [ "${CU_VERSION:-}" != "cpu" ] ; then
+    if [[ ${#CU_VERSION} -eq 4 ]]; then
+        CUDA_VERSION="${CU_VERSION:2:1}.${CU_VERSION:3:1}"
```

Comment:
Nit: we don't support CUDA-9 nor CUDA-8 now, why do we still adding this logic?

Reply:
Sure, I'll remove them